### PR TITLE
Create New-Programming-Languages-Showcase.yml

### PR DESCRIPTION
**Pull request by multikitty**

### CODE REVIEW

**Comment by multikitty** [comment]
what is wrong?

File: _data/projects/New-Programming-Languages-Showcase.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - showcase
+  - brief-history
+  - development-details
+  - future-technologies
+  - supporter
+  - advantages
+  - news
+upforgrabs:
+  name: Wecome, contributors!
+  link: https://github.com/multikitty/New-Programming-Languages-Showcase/issues/1
```
Comment (shiftkey) [inline _data/projects/New-Programming-Languages-Showcase.yml]: This should be a URL to a label in the repository, matching the `name` value defined above.

Looking at the contents of that issue I think either `help wanted` or `good first issue` are what you could use here:

```suggestion
  name: help wanted
  link: https://github.com/multikitty/New-Programming-Languages-Showcase/labels/help%20wanted
```

Comment (multikitty) [reply _data/projects/New-Programming-Languages-Showcase.yml]: Thanks for your suggestion.

**Comment by multikitty** [comment]
Hello.
Why is my pull request still pending?
Is there any issue?

**Comment by multikitty** [comment]
Hi, @shiftkey .
Could you please review my pull request again?
What hasn't gone right yet?

Thanks

**Comment by haneydogz8888** [comment]
Thanks for your email
On Tue, May 30, 2023, 2:22 AM multikitty ***@***.***> wrote:




Hi, @shiftkey .
Could you please review my pull request again?
What hasn't gone right yet?

Thanks

File: _data/projects/New-Programming-Languages-Showcase.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  - showcase
+  - brief-history
+  - development-details
+  - future-technologies
+  - supporter
+  - advantages
+  - news
+upforgrabs:
+  name: help wanted
+  link: https://github.com/multikitty/New-Programming-Languages-Showcase/labels/help%20wanted
```
Comment (shiftkey) [inline _data/projects/New-Programming-Languages-Showcase.yml:16]: This link goes to a single welcome issue, which is itself empty. I'd love to see a list of tasks that contributors can work on here. Is that something you're interested in setting up before we merge this PR?

Comment (multikitty) [reply _data/projects/New-Programming-Languages-Showcase.yml:16]: Hello, I added some issues.
Can you please check and merge my pull request?

Thanks.

**Review by shiftkey — APPROVED** [review]

**Pull request merged**